api/dal.py

```
@@ -0,0 +1,63 @@
+from datetime import datetime, timedelta
+
+from sqlalchemy import insert, delete, update, select, desc
+
+from api.models import Weight
+from api.schema import WeightDataCLass
+from api.sql_alchemy import SingletonSession
+
+
+def create_weight(weight: WeightDataCLass):
+    try:
+        session = SingletonSession.get_session()
+        session.execute(
+            insert(Weight).values(
+                {
+                    Weight.id: weight.id,
+                    Weight.value: weight.weight,
+                    Weight.when: weight.time,
+                }
+            )
+        )
+        session.commit()
+    except:
```

Don't use bare Except, catch specific exceptionsin this case we are only expecting sqlalchemy exceptions